### PR TITLE
docs: add info on our partner program

### PR DESCRIPTION
**Pull request by nicks**
Hello @dbentley, @landism,

Please review the following commits I made in branch nicks/partner:

e604f806649912eb8202af9ae8629770f5b87db9 (2019-10-30 17:37:14 -0400)
docs: add info on our partner program

### CODE REVIEW

File: docs/partner_program.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+- Machine: 8c581ff2fc00c6a47ecbd50abe47fb40
+- Repo: 3QLdKIWhsYTCsPI0vtsx6Q==
+```
+
+All strings in analytics are hashed, to help protect your privacy and your code.
+We use the aggregate data so that both of us (our team and your dev-ops team) has
+an accurate picture of how your team is using Tilt and what parts of it are slow
+
+If this sounds interesting to you, please email our CEO
+[Dan](mailto:dan@tilt.dev) to get set up.
```
Comment (dbentley) [inline docs/partner_program.md]: Maybe add a subject line?

For bonus point, cloud.tilt.dev/signup would be awesome

Comment (dbentley) [reply docs/partner_program.md]: That is, it'd be cool if they can click a button (while logged-in) instead of having to email.

File: api/api.py [hunk 2]
```diff
@@ -669,3 +669,17 @@ def docker_prune_settings(disable: bool=True, max_age_mins: int=360, num_builds:
     interval_hrs: run a Docker Prune every ``interval_hrs`` hours (unless ``num_builds`` is set, in which case use the "prune every X builds" logic). Defaults to 1 hour
   """
   pass
+
+def analytics_settings(enable: bool) -> None:
```
Comment (dbentley) [inline api/api.py]: Think it's worth making a cloud module, to point where we're going?

Comment (nicks) [reply api/api.py]: i don't understand what you're asking

File: docs/partner_program.md (hunk 1 above)
Comment (nicks) [reply docs/partner_program.md]: this seems like it's not worth the effort. you can't currently log in to the docs page.

File: api/api.py (hunk 2 above)
Comment (dbentley) [reply api/api.py]: The same way we have an os module in the Tiltfile, have a module called cloud, that would configure the settings for how local Tilt interacts with Tilt Cloud.

Comment (nicks) [reply api/api.py]: hmmm...i don't see these as being related. the current api is for consistency with existing tilt commands, e.g., 'tilt analytics' -> 'analytics_settings', 'tilt docker-prune', -> 'docker_prune_settings', etc

File: docs/partner_program.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+teams at a time. We want to make sure this is a good use of time for both of
+us. Here's what we look for:
+
+- A team of at least five engineers that expect to use something like Tilt for their day-to-day development
+- A microservice app where Tilt would be able to add value
+- A dev or dev-ops person who's willing to evaluate Tilt and get it setup
+
+When we kick off a partnership, we ask that you:
+
+- Check in with us bi-weekly over video chat to help with the first month or two of onboarding
```
Comment (landism) [inline docs/partner_program.md]: one of my favorite words!

bi·week·ly
/bīˈwēklē/

adjective
done, produced, or occurring every two weeks or twice a week.

**Pull request merged**